example/App.tsx

```diff
@@ -0,0 +1,57 @@
+import React, {useEffect, useState} from 'react';
+import Navigator from './components/navigation/Navigator';
+import {InbrainContext} from './components/context/InbrainContext';
+import {RewardContext} from './components/context/RewardContext';
+import inbrain, {StatusBarConfig, NavigationBarConfig} from 'inbrain-surveys';
+
+const App = () => {
+  const [reward, setReward] = useState(0);
+
+  useEffect(() => {
+    // To test with Your account - replace the credentials below
+    const CLIENT_ID = '852dd4b7-1d05-4803-a1e3-037d0fcfe18f';
```

Comment:
Let's remove the client_id, secret and user_id from the sample app

Reply:
done, now we will use config file for this variables